doc/modules/changes/20240613_danieldouglas92

```diff
@@ -0,0 +1,4 @@
+Added: Functionality to visualize the fluid
+velocity when advecting the porosity with the Darcy velocity.
```

Comment:
```suggestion
velocity in models with a compositional field named 'porosity'. This postprocessor is independent of the method the porosity is actually advected with, i.e. it could be static, advected with the Darcy velocity, or advected according to the two-phase flow equations.
```